tests/monster_waves_tests/test_randomGenerator.cpp

```diff
@@ -0,0 +1,34 @@
+#include "RandomGenerator.hpp"
+#include <gtest/gtest.h>
+
+
+class RandomNumberTest : public ::testing::Test
+{
+protected:
+    RandomNumber randGen;
+};
+
+TEST_F(RandomNumberTest, TestInRangeINT)
```

Comment:
@JakubMastalski zachowuj konwencję nazewniczą. Jeżeli na dole masz TestMinMaxOrderFloat (choć nie wiem czy to dobra nazwa) to tu też trzymaj się tej konwencji
- [ ] ujednolicić nazwy testów